src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Bmi1.PlatformNotSupported.cs

```diff
@@ -17,6 +17,62 @@ public abstract class Bmi1
 
         public static bool IsSupported { get { return false; } }
 
+        public abstract class X64 : Bmi1
```

Comment:
Why are we using X64 despite some semantic ambiguities?

Reply:
Still waiting for more suggestions.

Reply:
Understand - naming is the easiest of problems 😄